packages/components/carousel/src/use-carousel.ts

```diff
@@ -210,11 +220,40 @@ export const useCarousel = (
     containerHeight.value = height
   }
 
+  function slotDefaultNode() {
```

Comment:
Maybe better to rename it to something like `PlaceholderItem`? `slotDefaultNode` is a little bit confusing.

Reply:
I agree with what @HeftyKoo said, the name is indeed a bit confusing. And I think it might be better to add a comment where used. @MrWeilian

Reply:
done